File: 1012 - area/area.java
import java.util.*;
import java.io.IOException;


public class Main{

    public static double areadotrianguloretangulo( double baseA, double alturaC  ){
        return ((baseA * alturaC) / 2.0 );
    }
    
    public static double areadocirculo( double raioC ){
        return ( 3.14159 * raioC * raioC);
    }
    
    public static double areadotrapezio( double baseA, double baseB, double alturaC )
    {
        return (((baseA + baseB) * alturaC ) / 2.0 );
    }
    
    public static double areadoquadrado( double ladoB )
    {
        return (ladoB * ladoB);
    }
    
    public static double areadoretangulo( double ladoA, double ladoB )
    {
        return (ladoA * ladoB);
    }

    public static void main (String[] args) throws IOException
    {
        Double a, b, c;
        String linha;
        String[] foo;
        Scanner input = new Scanner( System.in );
        linha = input.nextLine();
        foo = linha.split("\\s+");
        a = Double.parseDouble(foo[0]);
        b = Double.parseDouble(foo[1]);
        c = Double.parseDouble(foo[2]); 
        System.out.printf("TRIANGULO: %.3f\n", areadotrianguloretangulo(a, c));
        System.out.printf("CIRCULO: %.3f\n", areadocirculo(c));
        System.out.printf("TRAPEZIO: %.3f\n", areadotrapezio(a, b, c));
        System.out.printf("QUADRADO: %.3f\n", areadoquadrado(b));
        System.out.printf("RETANGULO: %.3f\n", areadoretangulo(a, b));   
        
    }
}

File: 1012 - area/area.py
"""
def areadotrianguloretangulo( baseA,  alturaC ):
    return ((baseA * alturaC) / 2.0 )

def areadocirculo( raioC ):
    return ( 3.14159 * raioC * raioC)

def areadotrapezio( baseA , baseB , alturaC ):
    return (((baseA + baseB) * alturaC ) / 2.0 )

def areadoquadrado( ladoB ):
    return (ladoB * ladoB)

def areadoretangulo( ladoA, ladoB ):
    return (ladoA * ladoB)

a, b, c = raw_input().split()

atr = areadotrianguloretangulo( float(a), float(c))
ac = areadocirculo( float(c) )
at = areadotrapezio( float(a), float(b), float(c))
aq = areadoquadrado( float(b) )
ar = areadoretangulo( float(a), float(b) )

print "TRIANGULO: {0:.3f}" . format(atr)
print "CIRCULO: {0:.3f}" . format(ac)
print "TRAPEZIO: {0:.3f}" . format(at)
print "QUADRADO: {0:.3f}" . format(aq)
print "RETANGULO: {0:.3f}" . format(ar)

"""

def areadotrianguloretangulo( baseA,  alturaC ):
    return ((baseA * alturaC) / 2.0 )

def areadocirculo( raioC ):
    return ( 3.14159 * raioC * raioC)

def areadotrapezio( baseA , baseB , alturaC ):
    return (((baseA + baseB) * alturaC ) / 2.0 )

def areadoquadrado( ladoB ):
    return (ladoB * ladoB)

def areadoretangulo( ladoA, ladoB ):
    return (ladoA * ladoB)

a, b, c = input().split()

atr = areadotrianguloretangulo( float(a), float(c))
ac = areadocirculo( float(c) )
at = areadotrapezio( float(a), float(b), float(c))
aq = areadoquadrado( float(b) )
ar = areadoretangulo( float(a), float(b) )

print ("TRIANGULO: {0:.3f}" . format(atr))
print ("CIRCULO: {0:.3f}" . format(ac))
print ("TRAPEZIO: {0:.3f}" . format(at))
print ("QUADRADO: {0:.3f}" . format(aq))
print ("RETANGULO: {0:.3f}" . format(ar))